frontend/src/components/Collections/components/Grid/components/CollectionsGrid/index.tsx

```diff
@@ -11,10 +11,15 @@ import CollectionRow from "../Row/CollectionRow";
 
 interface Props {
   collections: CollectionResponse[];
-  accessType: ACCESS_TYPE;
+  accessType?: ACCESS_TYPE;
+  displayVisibility?: VISIBILITY_TYPE;
```

Comment:
Optional display visibility will filter out visibilities that don't match the specified type
Access type is now also optional